### DISCUSSION TITLE
Add table of contents to every chunk when using chunkedhtml

### DISCUSSION THREAD

**Discussion by uob-hillmu** [jgm/pandoc#10130] [Q&A]
I'd like to be able to place a table of contents on the html page generated for each chunk when using chunkedhtml.  Each page's table of contents would ideally show just the contents of that chunk, but I'd be happy with the whole thing if that's what's possible.  Is it possible?

Thanks,

Matt

**Comment by jgm** [comment]
For the whole thing, use `--variable toc`.
